app/controllers/user/mygardens/plants_controller.rb

```diff
@@ -6,8 +6,23 @@ def index
 	def create
 		garden = Garden.find(params[:garden_id])
 		plant = Plant.find(params[:plant_id])
-		garden.plants << plant
-		garden.save
+    garden.plants << plant
+    garden.save
+    add_notification(garden, plant)
 		redirect_to("/user/mygardens/#{garden.id}")
-	end
+  end
+  
+  private 
+  
+  def add_notification(garden, plant)
+    title = plant.name.capitalize + " harvest time!"
+    description = "It's time to harvest your #{plant.name}"
+    harvest_date = plant.first_harvest.to_i.days.from_now.to_s[0..9]
```

Comment:
creates a new `event` based on the current date and the number of days to the `first harvest`